packages/fury-adapter-apib-serializer/test/fixtures/basic.apib

```diff
@@ -2,12 +2,8 @@ FORMAT: 1A
 
 # My API
 
-## Group A
-
 ### /message
 
 #### GET
 
-+ Response 200 (text/plain)
```

Comment:
With this particular change, I think we don't have any test coverage which has a content-type in response payload. Could you either restore it or create a specific test with request/response with content-types?

Reply:
`sample.apib` contains the test case.